tutorials/cos-getting-started-trial/cos-getting-started-trial.md

```diff
@@ -39,28 +40,9 @@ You are the owner of an e-commerce business or a web shop in the fashion industr
 [DONE]
 [ACCORDION-END]
 
-[ACCORDION-BEGIN [Step 2: ](Set up SAP Customer Order Sourcing via Booster)]
-
-1. Log on to the **SAP Business Technology Platform trial** cockpit and open your global account for the Cloud Foundry environment.
-
-2. In the navigation pane of the SAP Business Technology Platform cockpit, choose **Boosters**.
-
-3. Choose the booster **Prepare an Account for SAP Customer Order Sourcing**.
-
-4. Choose **Start**.
-
->In case you are not able to subscribe to **SAP Customer Order Sourcing**, you need to [assign quota to a Subaccount](https://help.sap.com/viewer/cd03af1a94a440f1b5dbc0dc50a0989b/Cloud/en-US/bdf64a959a5249cf88b414a7c01391df.html).
-
-The configuration and set-up of your new **SAP Customer Order Sourcing** trial system starts and finishes automatically.
-
-You want to learn how to setup SAP Customer Order Sourcing yourself? This tutorial can help: [Set Up SAP Customer Order Sourcing Manually](cos-manual-setup).
-
-[DONE]
-[ACCORDION-END]
-
 [ACCORDION-BEGIN [Step 3: ](Create a new environment)]
 
-Having created a service instance and the corresponding credentials, you can now upload master data into SAP Customer Order Sourcing. You do this via a REST API.
+After creating a service instance and the corresponding credentials, for more information see [Set Up SAP Customer Order Sourcing Manually](cos-manual-setup), you can upload master data into **SAP Customer Order Sourcing**. You do this via a REST API.
```

Comment:
Are you sure that you want this link to be in the middle of a sentence?

Reply:
Changed it.